### PR TITLE
Observability troubleshooting doc update

### PR DESCRIPTION
**Pull request by EmilyKatdell**
image

### CODE REVIEW

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits are required.

**Review by forrestxia — APPROVED** [review]

**Comment by forrestxia** [comment]
@taohe1012 @baoy1 fya

**Review by baoy1 — APPROVED** [review]

File: content/docs/observability/troubleshooting/_index.md [hunk 1]
```diff
@@ -242,4 +244,17 @@ The warning can arise when a self-signed certificate for otel-collector is issue
 [root@:~]$ kubectl describe pod -n $namespace $pod
 MountVolume.SetUp failed for volume "tls-secret" : secret "otel-collector-tls" not found
 Unable to attach or mount volumes: unmounted volumes=[tls-secret], unattached volumes=[vxflexos-config-params vxflexos-config tls-secret karavi-metrics-powerflex-configmap kube-api-access-4fqgl karavi-authorization-config proxy-server-root-certificate]: timed out waiting for the condition
-```
\ No newline at end of file
+```
+
+### Why do I see 'Failed calling webhook' error when reinstalling CSM for Observability?
+This warning can occur when a user uninstalls Observability by deleting the Kubernetes namespace before properly cleaning up by running `helm delete` on the Observability Helm installation. This results in the credential manager failing to properly integrate with Observability on future installations. The user may see the following error in the Grafana pod upon reinstallation:
```
Comment (taohe1012) [inline content/docs/observability/troubleshooting/_index.md:250]: In `The user may see the following error in the Grafana pod upon reinstallation`, the `Grafana pod` can be changed to  `module pods`

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by rajkumar-palani — COMMENTED** [review]
@EmilyKatdell - Can you please resolve if there is any queries and merge it?

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
There is open query in this PR, Can you please resolve asap?

File: content/docs/observability/troubleshooting/_index.md (hunk 1 above)
Comment (EmilyKatdell) [reply content/docs/observability/troubleshooting/_index.md]: Sure just updated the change

**Review by taohe1012 — APPROVED** [review]

**Review by baoy1 — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**